spec/rspec/matchers/expecteds_for_multiple_diffs_spec.rb

```diff
@@ -66,6 +66,9 @@ def create_matcher(stubs)
             message, differ, actual
           )).to eq(dedent <<-EOS)
             |a message
+            |Diff:
+    <diff is empty, do your objects produce identical inspect output?>
```

Comment:
The `|` is used to remove whitespace before it (consider it the edge of the terminal) so these can be formatted correctly.